src/xpcc/architecture/platform/driver/core/cortex/driver.xml

```diff
@@ -36,6 +36,8 @@
 		<!-- stm32f4 -->
 		<template device-family="f4" device-name="405|407|415|417|427|429|437|439|469|479" out="linkerscript.ld">linkerscript/stm32_dccm.ld.in</template>
 		<template device-family="f4" device-name="401|410|411|446" out="linkerscript.ld">linkerscript/stm32_ram.ld.in</template>
+		<!-- stm32l4 -->
+		<template device-family="l4" device-name="431|432|433|442|443|471|475|476|485|486" out="linkerscript.ld">linkerscript/stm32_ram.ld.in</template>
```

Comment:
Choosing via device id is broken anyways, in modm, we choose the linkerscript via memory map attributes. Let's worry about this when (if) we get there.